testsuite/unittests/test_gap_junction.sli

```diff
@@ -82,7 +98,7 @@ neuron_no_gap neuron_no_gap Connect
 0 /max_delay get /old_max Set
 0 /min_delay get /old_min Set
 
-neuron_gap neuron_gap 4.0 4.0 /gap_junction Connect
+neuron_no_gap neuron_no_gap 2.0 2.0 Connect
```

Comment:
This has to be a gap-junction connection. 
Like for example `neuron_gap neuron_gap /gap_junction Connect`

Gap-junction connections contribute to the calculation of `min_delay` and `max_delay` with delay `wfr_comm`. The idea here is to check that this works properly.